content/blog/2020/05/2020-05-31-machine-learning-plugin-community-bonding.adoc

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "Machine Learning Plugin"
+tags:
+- machinelearning
+- datascience
+- plugins
+- communitybonding
+- gsoc
+- gsoc2020
+author: loghijiaha
+---
+
+= Machine Learning Plugin
+
+== Journey of Google Summer of Code  2020 until Community Bonding
+
```

Comment:
```suggestion

= Machine Learning Plugin

== Journey of Google Summer of Code  2020 until Community Bonding

```

Will cause duplication of the header

Reply:
Should I remove both lines? or the first line?

Reply:
I would recommend removing both for better look and feel

Reply:
The second header could become a part of the blogpost title if you prefer